projects/aca-content/src/lib/components/toolbar/toggle-edit-offline/toggle-edit-offline.component.ts

```diff
@@ -39,15 +46,9 @@ import { MatIconModule } from '@angular/material/icon';
   selector: 'app-toggle-edit-offline',
   template: `
     <button mat-menu-item [attr.title]="nodeTitle | translate" (click)="onClick()">
-      <ng-container *ngIf="isNodeLocked">
-        <mat-icon>cancel</mat-icon>
-        <span>{{ 'APP.ACTIONS.EDIT_OFFLINE_CANCEL' | translate }}</span>
-      </ng-container>
-
-      <ng-container *ngIf="!isNodeLocked">
-        <mat-icon>edit</mat-icon>
-        <span>{{ 'APP.ACTIONS.EDIT_OFFLINE' | translate }}</span>
-      </ng-container>
+      <mat-icon *ngIf="isNodeLocked">cancel</mat-icon>
+      <mat-icon *ngIf="!isNodeLocked">edit</mat-icon>
```

Comment:
Can we merge two mat-icons into one? 

```suggestion
      <mat-icon >{{ isNodeLocked ? 'cancel' : 'edit' }}</mat-icon>
```

Reply:
@nikita-web-ua done, thank you for the suggestion